### PR TITLE
3d effect glass flip card ui added

### PR DESCRIPTION
**Pull request by Recdata**
added my card ui with effect of 3d glass flip. Hope you will like it.

### CODE REVIEW

File: index.html
Comment (Arman-ali-khan-786) [inline index.html:1]: please add your name and correct Ui number

**Review by Arman-ali-khan-786 — CHANGES_REQUESTED** [review]
First of all, get assigned then make a pr or else I won't accept it...
Screenshot (682)

add your name and do change the Ui number.

**Comment by Recdata** [comment]
ok i will do the changes. please assign me to add my card ui.
Thanks

**Review by Arman-ali-khan-786 — CHANGES_REQUESTED** [review]
everything looks good i will merge it just pull the current changes and then make a pr again okk...

**Pull request merged**